src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionView.tsx

```diff
@@ -39,7 +38,7 @@ const ProposedSolutionView: React.FC<ProposedSolutionViewProps> = ({
           }}
           sx={{ position: 'absolute', right: 75 }}
         >
-          <FontAwesomeIcon icon={faTrash} size="lg" data-testid={'deleteIcon'} />
+          <DeleteIcon data-testid="DeleteIcon" />
```

Comment:
Don't know if you need the data testid. Take it out and see if the tests still pass, if they fail then put it back

Reply:
Yea, you're right! we just removed it, tested it, and it works.☠️☠️☠️ - Chloe